tempesta_fw/sock.c

```diff
@@ -482,15 +485,39 @@ ss_tcp_process_data(struct sock *sk)
 		skb_dst_drop(skb);
 		skb->dev = NULL;
```

Comment:
It appears that `skb->dev = NULL` is always done by the Linux stack in `tcp_v4_do_rcv()`: http://lxr.free-electrons.com/source/net/ipv4/tcp_ipv4.c?v=4.1#L1626